### ISSUE TITLE
Removing Release Blocker Label

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#4674]
I'm intending to remove the label, instead going forward use the "top-10" category in bug triage project.

Reasons:
- we do not need multiple ways to "add priority"
- noting something as a 'release blocker' introduces needless debate of whether a problem is truly new or not and whether the impact is severe/widespread.
- The big red label and attaching to a PR feels like a personal ding, whether intended or not.
- It's pretty subjective to add 'release blocker', it's the QA group that says when we can release and when we cannot.

**Comment by ron-murhammer** [comment]
I don't have a strong opinion on what its called so that's fine with me. Either way we need some way to flag high priority issues so that they are hopefully addressed soon and so we do a review of any that are still open before releasing.

Only thing I'll add is that the 'QA group' has mostly become @prastle, @panther2, and myself (thanks fellas). So mostly I make a judgement based on their feedback and my own testing on whether to release. Definitely would be nice to have more testers/input but that's where we are at.

**Comment by DanVanAtta** [comment]
Thanks @ron-murhammer - good feedback. Thinking about it we need then I think to discuss this from a different angle, notably how we define high-impact issues and how we want to track them.

**Comment by DanVanAtta** [comment]
The context of 'removing' a single label is not appropriate for what really needs to be discussed. I opened:
https://github.com/triplea-game/triplea/issues/4678 for us to discuss specifically how we define priority and how to track it. 

TL;DR: we can perhaps drop the 'bug triage' project and would just need to work out 'high priority' labels as a team. The biggest complication is low-priority issues, for dev reported I think we can find those a new home perhaps, for community report low-priority we would admit defeat (we have plenty of high and normal priority to fix) and just close.
